xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-test/xwiki-platform-appwithinminutes-test-docker/src/test/it/org/xwiki/appwithinminutes/test/ui/UserClassFieldIT.java

```diff
@@ -253,54 +236,106 @@ public void testSaveAndInitialSelection()
 
         userPicker = new SuggestClassFieldEditPane("user1").getPicker();
         selectedUsers = userPicker.getSelectedSuggestions();
-        Assert.assertEquals(3, selectedUsers.size());
+        assertEquals(3, selectedUsers.size());
         assertUserSuggestion(selectedUsers.get(2), "foobar", "foobar", null);
-        Assert.assertEquals(Arrays.asList("XWiki.tmortagne", "XWiki.Admin", "foobar"), userPicker.getValues());
+        assertEquals(Arrays.asList("XWiki.tmortagne", "XWiki.Admin", "foobar"), userPicker.getValues());
 
         // Delete the fake user.
         selectedUsers.get(2).delete();
-        Assert.assertEquals(2, userPicker.getSelectedSuggestions().size());
+        assertEquals(2, userPicker.getSelectedSuggestions().size());
 
         // Delete all selected users.
         userPicker.clearSelectedSuggestions();
         editor.clickSaveAndContinue();
         editor.clickCancel().edit();
 
         userPicker = new SuggestClassFieldEditPane("user1").getPicker();
-        Assert.assertEquals(0, userPicker.getSelectedSuggestions().size());
-        Assert.assertTrue(userPicker.getValues().isEmpty());
+        assertEquals(0, userPicker.getSelectedSuggestions().size());
+        assertTrue(userPicker.getValues().isEmpty());
     }
 
     @Test
-    public void testApplicationEntry()
+    @Order(5)
+    void applicationEntry(TestUtils testUtils, TestReference testReference)
     {
+        ApplicationClassEditPage editor = goToEditor(testReference);
         // Create the application class.
         SuggestInputElement userPicker = new SuggestClassFieldEditPane(editor.addField("User").getName()).getPicker();
         userPicker.sendKeys("thomas").waitForSuggestions().sendKeys(Keys.ENTER);
         editor.clickSaveAndView();
 
+        String testClassName = testReference.getSpaceReferences().get(0).getName();
+        String testMethodName = testReference.getLastSpaceReference().getName();
         // Create the application entry.
         ClassSheetPage classSheetPage = new ClassSheetPage();
-        InlinePage entryEditPage = classSheetPage.createNewDocument(getTestClassName(), getTestMethodName() + "Entry");
+        testUtils.getDriver().findElement(By.cssSelector("input[value='Create the template']")).click();
+        testUtils.getDriver().findElement(By.cssSelector("div.box.warningmessage a")).click();
```

Comment:
To be moved to methods in `ClassSheetPage` if still needed once the default value not displayed issue is solved.